client/components/ReviewList.jsx

```diff
@@ -7,7 +7,7 @@ import { isProduction } from '../utils.js';
 
 const ReviewList = props => {
   return (
-     <div>
+      <div className={styles.reviewsMain}>
```

Comment:
at top of your app you can create a main class..its like namespacing your entire app's style !